README.md

```diff
@@ -1,54 +1,43 @@
 # Broadside [![Build Status](https://travis-ci.org/lumoslabs/broadside.svg?branch=master)](https://travis-ci.org/lumoslabs/broadside)
-A command-line tool for deploying applications on AWS EC2 Container Service (ECS)
-
-This tool is primarily intended for use with ruby applications.
 
+A command-line tool for deploying applications on AWS EC2 Container Service (ECS)
 
 ## Overview
 Amazon ECS presents a low barrier to entry for production-level docker applications. Combined with ECS's built-in blue-green deployment, Elastic Load Balancers, Autoscale Groups, and CloudWatch, one can set up a robust cluster that can scale to serve any number of applications in a short amount of time. Broadside seeks to leverage these benefits and improve the deployment process for developers.
 
-Broadside offers a simple command-line interface to perform deployments on ECS. It does not attempt to handle operational tasks like infrastructure setup and configuration, which are better suited for tools like [terraform](https://www.terraform.io/). This allows applications using broadside to employ a clean configuration file that looks something like:
+Broadside offers a simple command-line interface to perform deployments on ECS. It does not attempt to handle operational tasks like infrastructure setup and configuration, which are better suited for tools like [terraform](https://www.terraform.io/).
+
+Applications using broadside employ a configuration file that looks something like:
 
 ```ruby
 Broadside.configure do |config|
   config.application = 'hello_world'
   config.docker_image = 'lumoslabs/hello_world'
-  config.ecs.cluster = 'micro-cluster'
+  config.ecs.cluster = 'production-cluster'
   config.targets = {
     production_web: {
       scale: 7,
       command: ['bundle', 'exec', 'unicorn', '-c', 'config/unicorn.conf.rb'],
-      env_file: '../.env.production'
+      env_file: '.env.production'
       predeploy_commands: [
-        Broadside::Predeploy::RAKE_DB_MIGRATE,     # RAKE_DB_MIGRATE is just a constant for your convenience
-        ['bundle', 'exec', 'rake', 'data:migrate']
+        Broadside::Predeploy::RAKE_DB_MIGRATE, # RAKE_DB_MIGRATE is just a constant for your convenience
+        ['bundle', 'exec', 'rake', 'assets:precompile']
```

Comment:
nothing; just thought it was a more universal example but i'm happy to put it back.